deploy/manifests/crds.yaml

```diff
@@ -1,4 +1,5 @@
 ---
+---
```

Comment:
Something went wrong here.

Reply:
Yeah i have the same with HBase, not sure where this is coming from?

Reply:
we want to get rid of these anyway.